pkg/resource/testdata/TestStatefulSetBuilder/default_insecure_statefulset.golden

```diff
@@ -46,7 +46,7 @@ spec:
             resourceFieldRef:
               divisor: 1Mi
               resource: limits.memory
-        image: cockroachdb/cockroach:v20.2.7
+        image: cockroachdb/cockroach:v21.1.0
```

Comment:
Should be v21.2.0?

Reply:
No, we have upgraded the cockroach binary to `v21.2.0` in operator pod, however logging APIs are supported from `v21.1.0`, hence in test cases I have added the minimum supported version. This way we can test the statefulset yaml is properly generated on this minimum version